packages/core/useKeyModifier/index.ts

```diff
@@ -34,7 +34,7 @@ export function useKeyModifier<Initial extends boolean | null>(modifier: KeyModi
     initial = null,
   } = options
 
-  const state = deepRef(initial) as Ref<boolean>
+  const state = shallowRef(initial) as ShallowRef<boolean>
```

Comment:
Similar I guess this one can be `shallowRef<boolean>(initial)`? Without the cast